docs/source/appendices/install-os-level-deps.md

```diff
@@ -0,0 +1,17 @@
+# How to Install OS-Level Dependencies
+
+BigchainDB Server has some OS-level dependencies that must be installed.
+
+On Ubuntu 14.04, we found that the following was enough:
+```text
+sudo apt-get update
+sudo apt-get install g++ python3-dev
+```
+
+On Fedora 23, we found that the following was enough (tested in February 2015):
```

Comment:
I recently did the same in a new installation of Fedora 24 at home so I guess we could change:
- `Fedora 23 -> Fedora 23/24`
- remove `(tested in February 2015)`